Adyen/UI/Form/Items/SelectableFormItem/SelectableFormItemView.swift

```diff
@@ -0,0 +1,179 @@
+//
+// Copyright (c) 2024 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import UIKit
+
+/// A view representing a selectableFormItem item.
+@_spi(AdyenInternal)
+public final class SelectableFormItemView: FormItemView<SelectableFormItem> {
+    private let imageLoader: ImageLoading = ImageLoaderProvider.imageLoader()
+    private var imageLoadingTask: AdyenCancellable?
```

Comment:
We should make sure the `imageLoadingTask` is cancelled when a new task is started - You can checkout the example in the `ListItemView` 👍

Reply:
6af2d17dede672e40b74333a11030270d8564c46